src/hpr_routing.erl

```diff
@@ -97,95 +103,159 @@ find_routes({uplink, {_Type, DevAddr}}, PacketUp) ->
     ok = hpr_metrics:observe_find_routes(Time),
     Results.
 
+-spec start_crawl_routing() -> pid().
+start_crawl_routing() ->
```

Comment:
Echo Jeff's comment - typically you would use a spawn_monitor here and if the pid dies, you'd get a message to this gen server to do something about it. If it's ok this dies and gets restarted randomly, maybe it doesn't matter.